src/Page/Definition.elm

```diff
@@ -27,4 +30,30 @@ update msg model =
 
 view : Model -> Html Msg
 view model =
-    div [] [ text "Definition page" ]
+    div []
+        [ header []
+            [ h1 [ css [ pageHeading ] ] [ text "[cCc] What is Economic Abuse?" ]
+            , p [ css [ pageDescription ] ] [ text "[cCc] A concise definition of economic abuse in accessible language, and that spans a few sentences so that a small intro paragraph doesn't look strange here.", text " ", a [ href "/[cCc]somewhere" ] [ text "[cCc] more information" ] ]
+            ]
+        , ul []
+            [ li [ css [ category ] ] [ h2 [] [ text "[cCc] A Category" ] ]
+            , li [ css [ category ] ] [ h2 [] [ text "[cCc] A Category" ] ]
+            , li [ css [ category ] ] [ h2 [] [ text "[cCc] A Category" ] ]
+            ]
+        ]
+
+
+category : Style
+category =
+    Css.batch
+        [ margin2 (rem 2) zero ]
+
+
+pageDescription : Style
+pageDescription =
+    Css.batch
+        [ color colours.purple
+        , fontFamilies [ "Raleway", sansSerif.value ]
```

Comment:
Should Raleway only be for the h1 / h2 etc or is is meant for use in contextual stuff? Isn't this 'description' actually the definition? Our other pages do't have descriptions so might be more clear as 'intro' or 'definition'

Not really an issue.

Reply:
The style guide says that Raleway should be used for page intro paragraphs, which this feels like it is. I think we may want to soften it a bit later, but we'll probably know that when there's more on the page.